OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -29,14 +29,18 @@ public virtual void Init(ModData modData, Dictionary<string, string> info)
 			ModData = modData;
 		}
 
+		bool initialized;
+
 		public virtual void Display()
 		{
-			if (Game.Renderer == null)
+			if (Game.Renderer == null || initialized)
 				return;
 
 			// Draw a black screen
 			Game.Renderer.BeginUI();
 			Game.Renderer.EndFrame(new NullInputHandler());
+
+			initialized = true; // PERF: draw the screen only once
```

Comment:
Style nit: Comments should go on their own line above the line commented on.

Reply:
Moved.